package.json

```diff
@@ -41,9 +41,12 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm test"
+      "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "*.{js,jsx,md}": ["npm test"]
```

Comment:
Although I'm not sure if we should run tests here as well. Perhaps just linting would be enough?

Reply:
What does `test` run? :D

Reply:
LOL 😄 I guess I was thinking about core repo while writing this